.github/workflows/upload-assets-to-cdn.yml

```diff
@@ -41,8 +41,8 @@ jobs:
 
       - name: "Asset upload to production CDN S3 bucket"
         run: |
-          aws s3 cp --cache-control 'public,max-age=86400' --recursive assets/product-icons --exclude "*" --include "*.svg" --include "*.png" s3://fxa-content-cdn-prod-distbucket-gqg70i8xqycy/product-icons
-          aws s3 cp --cache-control 'public,max-age=86400' --recursive --content-disposition attachment --exclude "*" --include "*.pdf"  assets/legal s3://fxa-content-cdn-prod-distbucket-gqg70i8xqycy/legal
+          aws s3 sync --cache-control 'public,max-age=86400' --exclude "*" --include "*.svg" --include "*.png" assets/product-icons  s3://fxa-content-cdn-prod-distbucket-gqg70i8xqycy/product-icons
+          aws s3 sync --cache-control 'public,max-age=86400' --exclude "*" --include "*.pdf" --content-disposition attachment   assets/legal s3://fxa-content-cdn-prod-distbucket-gqg70i8xqycy/legal
```

Comment:
Has the business signed off on this? If they want old docs to still link, `cp` was the right thing. I was just asking a question :)